io.elementary.code.json

```diff
@@ -0,0 +1,247 @@
+{
+	"app-id": "io.elementary.code",
+	"runtime": "org.gnome.Platform",
+	"runtime-version": "3.24",
+	"sdk": "org.gnome.Sdk",
+	"command": "scratch-text-editor",
+	"rename-desktop-file": "org.pantheon.scratch.desktop",
+	"rename-appdata-file": "org.pantheon.scratch.appdata.xml",
+	"finish-args": [
+		"--filesystem=host",
+		/* Wayland and X11 access */
```

Comment:
Thanks for your explanation, @manuq. I agree with @TingPing, we should avoid them.

-In this case, they doesn't look pretty in Github ;)